modules/building/pages/accessing-private-images.adoc

```diff
@@ -0,0 +1,66 @@
+= Accessing private image repositories
+
+When the image repository visibility is set to `private` (the default), you need to authenticate before you can pull images. Konflux provides a proxy RBAC image server that allows authorized users to pull private images.
+
+== Getting registry login credentials via UI
+
+To access private images locally:
+
+. Navigate to the *Component details* page for your component
+. In the *Registry login information* section, click the OAuth URL link to open the proxy registry server authentication page
+. Copy the token or the podman login command provided on that page
+. Run the login command in your terminal
+
+After logging in, you can pull images from your private repositories on your local machine.
+
+== Creating service accounts for external systems
+
+If you need to pull container images or OCI artifacts from your Konflux component repositories in external systems outside of Konflux (such as Testing Farm or other CI/CD environments), you can create a Kubernetes service account with the necessary permissions. This allows automated systems to authenticate and pull your private images without requiring interactive login.
+
+=== Prerequisites
+
+You need the following information from the *Component details* page:
+
+* Registry URL
+* Username
+* Authentication token (obtained via the OAuth URL link in the *Registry login information* section)
```

Comment:
How does rotation and invalidation of the work?

Reply:
If the token has expired, the login will fail, and the user will need to revisit the OAuth URL to obtain a new valid token. This is a normal flow, so I don’t think we need to explicitly document it here.

Are you suggesting us providing brief guidance, such as “If you’re unable to log in, please contact Konflux user support”?